packages/expo-document-picker/src/ExpoDocumentPicker.web.ts

```diff
@@ -59,7 +59,7 @@ function readFileAsync(targetFile: File, base64: boolean = true): Promise<Docume
     const mimeType = targetFile.type;
     if (!base64) {
       resolve({
-        uri: targetFile.webkitRelativePath,
```

Comment:
This was always an empty string and never returned anything, since this only work with the webkitdirectory directive on input.